submissions/sprint_a/__init__.py

```diff
@@ -0,0 +1,14 @@
+def configure_app():
+    import os
+    from dotenv import load_dotenv
+    from flask import Flask, jsonify
```

Comment:
You don't need to import jsonify here. It's a flask module but not part of flask configuration. You import jsonify anywhere in an app and it's fine.